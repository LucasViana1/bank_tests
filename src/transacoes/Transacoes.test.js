import React from 'react'
import { render } from '@testing-library/react';
import Transacoes from './Transacoes';


describe('Componente de listagem de transações', ()=>{
  it('O snapshot do componente deve permanecer sempre o mesmo', ()=>{
    const transacoesProps = {
      transacoes:[
        {
          id: 1,
          tipo: 'deposito',
          valor: '1000.00',
          data: '01/11/2020',
        },
        {
          id:2,
          tipo: 'saque',
          valor: '200.00',
          data: '03/11/2020',
        },
      ]
    }

    const { container } = render(
      <Transacoes {...transacoesProps} />
    )

    expect(container.firstChild).toMatchSnapshot();
  })
})
