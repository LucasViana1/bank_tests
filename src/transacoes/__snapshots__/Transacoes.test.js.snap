// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de listagem de transações O snapshot do componente deve permanecer sempre o mesmo 1`] = `
<div
  data-testid="transacoes"
>
  <div
    class="transacao-container"
  >
    <p>
      01/11/2020
    </p>
    <p />
    <p>
      R$ 
      1000.00
    </p>
  </div>
  <div
    class="transacao-container"
  >
    <p>
      03/11/2020
    </p>
    <p />
    <p>
      R$ 
      200.00
    </p>
  </div>
</div>
`;
